### ISSUE TITLE
emesene-2.12.9 installs .desktop files that do not pass validation

### ISSUE THREAD

**Issue by pacho2** [emesene/emesene#1530]
desktop-file-validate outputs:
/usr/lib64/python2.7/site-packages/emesene/data/share/applications/emesene.desktop: error: file contains group "NewInstance Shortcut Group", but groups extending the format should start with "X-"
        /usr/share/applications/emesene-2.desktop: error: file contains group "NewInstance Shortcut Group", but groups extending the format should start with "X-"

Thanks

**Comment by KinG-InFeT** [comment]
what operating system are you using? that version?

**Comment by pacho2** [comment]
Gentoo Linux,

**Comment by KinG-InFeT** [comment]
windows manager? version?

**Comment by pacho2** [comment]
Gnome 2.32... but it is not related at all with freedesktop specs and
desktop-file-validate

2012/10/2 Vincenzo notifications@github.com

**Comment by KinG-InFeT** [comment]
probably for gnome 2.x file. desktop is no longer valid since a WM dated and the gentoo ebuild file is configured differently for WM of the new generation.
You should also open a track in bugs.gentoo.org and see what answer.

**Comment by Sbte** [comment]
@KinG-InFeT please don't interfere with things like these. You could have just run desktop-file-validate yourself to see that the desktop file isn't validated.

@pacho2 good catch, but this is the way suggested by the Unity people. I could add "X-" to the shortcut group name, but is that really necessary? This probably holds for most applications that implement a Unity shortcut menu.

**Comment by KinG-InFeT** [comment]
@Sbte sorry, I just wanted to lend a hand

**Comment by pacho2** [comment]
Probably needs to be fixed in spec then like was recently done with "GNOME3":
https://bugs.freedesktop.org/show_bug.cgi?id=55483

But, as this could end up on an infinite flame due usual conflicts with Unity/Ubuntu, could you contact that Unity people to ask them to report the problem?

**Comment by KinG-InFeT** [comment]
Vincent Untz 2012-10-03 13:56:26 UTC

Fixed in git.

**Comment by Sbte** [comment]
@pacho2 I don't think it can be fixed like that, because everyone can use custom names for the Unity launcher shortcuts. Anyway, this will be fixed when we move to Gtk3, because then we can make use of the Unity Python API. Since this will most likely be in the next release or so, I don't think this is something worth fixing.
